### PR TITLE
Use Subversion version specified in BOM

### PR DESCRIPTION
**Pull request by basil**
The Subversion plugin update is needed to adapt to the Commons Digester removal in PCT tests for 2.297 and later.

### CODE REVIEW

**Review by jetersen — APPROVED** [review]

**Review by bitwiseman — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -23,12 +23,12 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (jetersen) [inline pom.xml:26]: Why this change? http://maven.apache.org/maven-v4_0_0.xsd
Seems to redirect to http://maven.apache.org/xsd/maven-4.0.0.xsd

Comment (basil) [reply pom.xml:26]: Being consistent with the archetype. If we want to use a different URL, the archetype should be updated.

**Comment by basil** [comment]
@car-roll Friendly ping that it would be nice to get this merged and released at some point in the next few weeks (no rush) to get BOM testing back in shape for weeklies.

**Review by car-roll — APPROVED** [review]
